ENGDOCS-1875 (#18788)
docker/docs: content/desktop/hardened-desktop/settings-management/configure.md

    "locked": false,
    "sbomIndexing": true,
    "useBackgroundIndexing": true
  },
  "allowExperimentalFeatures": {
    "locked": true,

>     "value": true

Reviewer: I believe here we should give the example of value=false which is in sync with the "Turn off beta and experimental features" addition in _index.md

Also, all the other examples here have "locked": false. Shall this be the same, for consistency?

Also, for consistency, can we have the order of locked, value everywhere? as in
```
"x": {
    "locked":y,
    "value":z:
  },
```
for example"extensionsEnabled" and "blockDockerLoad" have it the other way around

  },
  "allowBetaFeatures": {
    "locked": true,

>     "value": true

Reviewer: same here,  suggested "value": false

|`analyticsEnabled`|  |If `value` is set to false, Docker Desktop doesn't send usage statistics to Docker. |
|`extensionsEnabled`|  |If `value` is set to false, Docker extensions are disabled. |
|`scout`|| Setting `useBackgroundIndexing` to `false` disables automatic indexing of images loaded to the image store. Setting `sbomIndexing` to `false` prevents the manual indexing triggered by inspecting an image in Docker Desktop.<br><br>**Note**: Users can still use the `docker scout` CLI commands to index images, even if indexing is disabled in Settings Management. |

> | `allowExperimentalFeatures`| | If `locked` is set to `true`, users are no longer able to access or use experimental features.|

Reviewer: ```suggestion
| `allowExperimentalFeatures`| | If `value` is set to `false`, the experimental features are disabled.|
```

Reviewer: As with the other admin settings, here we only need to describe what the setting controls, as the locking behaviour is described at the top of the page, line 46

> | `allowBetaFeatures`| | If `locked` is set to `true`, users are no longer able to access or use beta features.|

Reviewer: ```suggestion
| `allowBetaFeatures`| | If `value` is set to `false`, the beta features are disabled.|
```

> | `blockDockerLoad` | | If `value` is set to `true`, users are no longer able to run [`docker load`](../../../engine/reference/commandline/load.md).|

Reviewer: @gabriellavengeo please review this

Reviewer 2: Yeah, that's fine, or alternatively say that they should get an error if they try to
